notebooks/image_models/solutions/create_tfrecords_at_scale.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>I was wondering if we should provide instructions in markdown/ split the code into multiple cells to show how each part works. Since it's a .py file, it's pretty hard to debug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>+1 I'd break this code in chunk to be run in cells prefixed by explanations in markdown.</p><p>As it is, it seems too bulky for a first dive into dataflow. </p>